conf/config.level0.neon

```diff
@@ -27,6 +27,7 @@ rules:
 	- PHPStan\Rules\Functions\InnerFunctionRule
 	- PHPStan\Rules\Functions\NonExistentDefinedFunctionRule
 	- PHPStan\Rules\Functions\PrintfParametersRule
+	- PHPStan\Rules\Functions\RandomIntParametersRule
```

Comment:
* This should be in level 5 for consistency with other type-accepting rules.
* New rules can only be added to bleeding edge. E.g.:
  1) create a new feature toggle in config.level5.neon under `featureToggles` option key with `false` value.
  2) Override the parameter value with `true` in bleedingEdge.neon.
  3) Do not put the rule in the `rules` section in config.level5.neon, but put it under `conditionalTags` like this: https://github.com/phpstan/phpstan-src/commit/25101c464ca0f8b9e3a552fd24dbc01a201787f8#diff-bbff7dc80cd298341933c387916c4e21 (you also need to register the rule in `services` section but without the rule tag). The linked commit basically shows all three points :)

Reply:
Hopefully I've done this correctly.